examples/unmanaged_device.rb

```diff
@@ -1,29 +1,29 @@
-# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed
-# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-# CONDITIONS OF ANY KIND, either express or implied. See the License for the
-# specific language governing permissions and limitations under the License.
-
-my_client = {
-  url: '',
-  user: '',
-  password: ''
-}
-
-oneview_unmanaged_device 'UnmanagedDevice1' do
-  client my_client
-  data(
-    model: 'Procurve 4200VL',
-    deviceType: 'Server'
-  )
-end
-
-oneview_unmanaged_device 'UnmanagedDevice1' do
-  client my_client
-  action :delete
-end
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+my_client = {
+  url: '',
+  user: '',
+  password: ''
+}
+
+oneview_unmanaged_device 'UnmanagedDevice1' do
+  client my_client
+  data(
+    model: 'Procurve 4200VL',
+    deviceType: 'Server'
+  )
+end
+
+oneview_unmanaged_device 'UnmanagedDevice1' do
+  client my_client
+  action :delete
+end
```

Comment:
This "change" is just line-ending fixes. @anazard , you may want to check your Git config settings. I always set core.autocrlf to input.